src/test/java/uk/gov/ons/ctp/response/action/service/decorator/PartyActionRequestDecoratorTest.java

```diff
@@ -1,131 +1,131 @@
-
-    final String respondentStatus =
-        this.partyActionRequestDecorator.parseRespondentStatuses(partyList);
-
-    assertEquals(ActionProcessingService.ACTIVE, respondentStatus);
-  }
-
-  @Test
-  public void testParseRespondentStatusesEmpty() {
-    final List<PartyDTO> partyList = new ArrayList<>();
-    final String respondentStatus =
-        this.partyActionRequestDecorator.parseRespondentStatuses(partyList);
-
-    assertEquals(null, respondentStatus);
-  }
-
-  @Test
-  public void testGetEnrolmentStatusEnabled() {
-    final PartyDTO partyDTO = partyDTOs.get(0);
-    assertEquals(
-        ActionProcessingService.ENABLED,
-        this.partyActionRequestDecorator.getEnrolmentStatus(partyDTO));
-  }
-
-  @Test
-  public void testGetEnrolmentStatusPending() {
-    final PartyDTO partyDTO = partyDTOs.get(1);
-    assertEquals(
-        ActionProcessingService.PENDING,
-        this.partyActionRequestDecorator.getEnrolmentStatus(partyDTO));
-  }
-
-  @Test
-  public void testGetEnrolmentStatusDefault() {
-    final PartyDTO partyDTO = partyDTOs.get(2);
-    assertEquals(null, this.partyActionRequestDecorator.getEnrolmentStatus(partyDTO));
-  }
-
-  @Test
-  public void testGetEnrolmentStatusNoEnrolments() {
-    final PartyDTO partyDTO = partyDTOs.get(2);
-    partyDTO.setAssociations(null);
-    assertEquals(null, this.partyActionRequestDecorator.getEnrolmentStatus(partyDTO));
-  }
-
-  @Test
-  public void testGenerateTradingStyle() {
-    final Attributes businessAttributes = new Attributes();
-    businessAttributes.setTradstyle1("TRADSTYLE1");
-    businessAttributes.setTradstyle2("TRADSTYLE2");
-    businessAttributes.setTradstyle3("TRADSTYLE3");
-
-    final String generatedTradingStyle =
-        this.partyActionRequestDecorator.generateTradingStyle(businessAttributes);
-    final String expectedTradingStyle = "TRADSTYLE1 TRADSTYLE2 TRADSTYLE3";
-
-    assertEquals(expectedTradingStyle, generatedTradingStyle);
-  }
-
-  @Test
-  public void testGenerateTradingStyleWithEmptyValues() {
-    final Attributes businessAttributes = new Attributes();
-
-    final String generatedTradingStyle =
-        this.partyActionRequestDecorator.generateTradingStyle(businessAttributes);
-    final String expectedTradingStyle = "";
-
-    assertEquals(expectedTradingStyle, generatedTradingStyle);
-  }
-
-  @Test
-  public void testGenerateTradingStyleWithSubsetOfTradingStyles() {
-    final Attributes businessAttributes = new Attributes();
-    businessAttributes.setTradstyle1("TRADSTYLE1");
-    businessAttributes.setTradstyle3("TRADSTYLE3");
-
-    final String generatedTradingStyle =
-        this.partyActionRequestDecorator.generateTradingStyle(businessAttributes);
-    final String expectedTradingStyle = "TRADSTYLE1 TRADSTYLE3";
-
-    assertEquals(expectedTradingStyle, generatedTradingStyle);
-  }
-}
+// package uk.gov.ons.ctp.response.action.service.decorator;
```

Comment:
I presume you're planning on getting rid of this class before you merge it?

Reply:
Working on fixing these tests today, will get them up tomorrow